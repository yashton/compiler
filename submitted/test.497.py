if (x== -1):
  return 1.j

 elif x ==.1e000:

     return 1
else
  return 1