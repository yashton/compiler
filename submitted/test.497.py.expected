(KEYWORD if)
(PUNCT "(")
(ID "x")
(PUNCT "==")
(PUNCT "-")
(LIT 1)
(PUNCT ")")
(PUNCT ":")
(NEWLINE)
(INDENT)
(KEYWORD return)
(LIT +1.i)
(NEWLINE)
(DEDENT)
(ERROR "mis-indented program")
